app/presenters/job_presenter.rb

```diff
@@ -26,6 +26,17 @@ def list(jobs)
         single(job)
       end.compact
     end
+    
+    def list_by_cluster_id(jobs, cluster_id)
+      jobs.map do |job|
+        unless job['payload'].blank?
```

Comment:
Instead of increasing indentation, prefer guard clauses.
Also, are we sure that job['payload'] is always non-nil? `.blank?` does not exist as a method on `nil`.

How about this:
`next unless job['payload'] && job['payload'].blank?`